test/e2e/dataplane/tcp_pod_to_service.go

```diff
@@ -15,8 +15,8 @@ var _ = Describe("[dataplane] Basic Pod to Service tests across clusters without
 		testPod2ServiceTCP(f, framework.NonGatewayNode, framework.NonGatewayNode)
 	})
 
-	It("Should be able to perform a Pod to Service TCP connection and exchange data between different clusters NonGW to GW", func() {
-		testPod2ServiceTCP(f, framework.NonGatewayNode, framework.GatewayNode)
+	It("Should be able to perform a Pod to Service TCP connection and exchange data between different clusters GW to GW", func() {
```

Comment:
Thanks for addressing the other comments.

Lets validate GW to GW use-case before nonGW to nonGW (i.e., move this to line before 14). 
If GW to GW fails, then the second use-case is bound to fail.

Reply:
Just making a note on the Github interface that wasn't captured here (Sorry for the repetition)..

ginkgo runs the tests randomly so the order does not really matter.